views/cart.hbs

```diff
@@ -0,0 +1,101 @@
+{{!-- <section class="cart_Box">
+    <div class="container">
+        <div>
+            <h3>My Cart<span>(1 item)</span></h3>
+            <div class="cart_Items">
+                <ul>
+                    <li>
+                        <div class="items flex_box">
+                            <div class="img_wrap">
+                                <img
+                                    src="img/products/fruit-n-veg/apple.jpg"
+                                    alt="">
+                            </div>
+                            <div class="detail_wrap">
+                                <h5>Fruits & Vegetable</h5>
+                                <div>
+                                    <button class="btn">-</button>
```

Comment:
How will you attach a - or + event?

Reply:
Events are attached on the respective buttons.